src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1192,6 +1194,19 @@ public Builder shardPrioritization(ShardPrioritization shardPrioritization) {
             return this;
         }
 
+        /**
+         * Set KinesisProxy for the worker.
+         *
+         * @param kinesisProxy
+         *            KinesisProxy uses the AmazonKinesis client to get data from Kinesis or DynamoDBStreams
```

Comment:
Can you fix this comment, it would be confusing for the customers since KCL already has DDB endpoints. Can you remove DynaoDBStreams from the comment.